eemeter/models/temperature_sensitivity.py

```diff
@@ -78,43 +99,52 @@ def objective_function(param_array):
         return self.param_type(params)
 
     def transform(self, X, params):
+        # TODO: Is it supposed to produce a prediction?  Why not call it
```

Comment:
Did I mess up following the scikit learn API? Maybe I don't understand the difference between transform and predict. Seems reasonable to rename it if that would be help clarify.